public/app/features/library-panels/components/OpenLibraryPanelModal/OpenLibraryPanelModal.tsx

```diff
@@ -38,38 +39,54 @@ export function OpenLibraryPanelModal({ libraryPanel, onDismiss }: OpenLibraryPa
   };
 
   return (
-    <Modal title="View panel in dashboard" onDismiss={onDismiss} onClickBackdrop={onDismiss} isOpen>
+    <Modal
+      title={t('library-panels.modal.title', 'View panel in dashboard')}
+      onDismiss={onDismiss}
+      onClickBackdrop={onDismiss}
+      isOpen
+    >
       <div className={styles.container}>
         {connected === 0 ? (
-          <span>Panel is not linked to a dashboard. Add the panel to a dashboard and retry.</span>
+          <span>
+            <Trans i18nKey={'library-panels.modal.panel-not-linked'}>Add the panel to a dashboard and retry.</Trans>
+          </span>
         ) : null}
         {connected > 0 ? (
           <>
             <p>
-              This panel is being used in{' '}
+              <Trans i18nKey={'library-panels.modal.contianer-prefix'}>This panel is being used in </Trans>
               <strong>
-                {connected} {connected > 1 ? 'dashboards' : 'dashboard'}
+                {connected}{' '}
+                {connected > 1 ? (
+                  <Trans i18nKey={'library-panels.modal.contianer-dashboards'}>dashboards</Trans>
+                ) : (
+                  <Trans i18nKey={'library-panels.modal.contianer-dashboard'}>dashboard</Trans>
+                )}
               </strong>
-              .Please choose which dashboard to view the panel in:
+              <Trans i18nKey={'library-panels.modal.contianer-suffix'}>
+                Please choose which dashboard to view the panel in:
+              </Trans>
```

Comment:
there's some special handling for plural forms. you can use the `count` property, then i think we can replace all of this with:

```tsx
              <Trans i18nKey="library-panels.modal.body" count={connected}>
                This panel is being used in {{ count: connected }} dashboard.
                Please choose which dashboard to view the panel in:
              </Trans>
```
this will generate 2 translation strings, one for the singular and one for other, then manually edit these in the en-us and pseudo translation files. this is mentioned in the documentation [here](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#plurals)